.changeset/nice-feet-hope.md

```diff
@@ -0,0 +1,30 @@
+---
+'@backstage/create-app': patch
+---
+
+The `index.html` template of the app has been updated to use the new `config` global provided by the Backstage CLI.
+
+To apply this change to an existing app, make the following changes to `packages/app/public/index.html`:
+
+```diff
+-    <title><%= app.title %></title>
++    <title><%= config.getString('app.title') %></title>
+```
+
+```diff
+-    <% if (app.googleAnalyticsTrackingId && typeof app.googleAnalyticsTrackingId === 'string') { %>
```

Comment:
Oh, right. We should probably remove this in favor of the Analytics API.  ...I'll handle later in follow-up. 😅

Reply:
Yep wanted to talk to you about this, I'm thinking each of these different snippets would instead be part of separate analytics modules. Works much better with config schema and whatnot too. Main thing there is whether there's a good way to avoid the need for modifying index.html, but that can be handled per-module